### PR TITLE
feat(focus): new focus indicator component

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

File: src/sass/mixins/appearance/_mixins.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ $all-appearances: (
 
 %outlined {
 	#{variables.$appearance-color-fill}: transparent;
-	#{variables.$appearance-color-outline}: var(#{connotation.$vvd-color-connotation});
+	#{variables.$appearance-color-outline}: var(#{connotation.$vvd-color-connotation}-diverse);
```
Comment (rachelbt) [inline src/sass/mixins/appearance/_mixins.scss:26]: not sure this is the reason but now the outlined in disable mode is still primary color:
image

Comment (yinonov) [reply src/sass/mixins/appearance/_mixins.scss:26]: right! it's no longer border but a box-shadow

File: src/components/badge/badge.spec.ts [hunk 2]
```diff
@@ -15,14 +15,22 @@ describe('vwc-badge', () => {
...
+			expect(element.iconTrailing)
+				.toBeFalsy();
+			expect(element.connotation)
+				.toBeUndefined();
+			expect(element.shape)
+				.toBeUndefined();
+			expect(element.appearance)
+				.toBeUndefined();
+			expect(element.size)
+				.toBeUndefined();
```
Comment (YonatanKra) [inline src/components/badge/badge.spec.ts]: Is that how we want it to be? Why not one liner?

File: src/components/focus/README.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# vwc-focus
```
Comment (YonatanKra) [inline src/components/focus/README.md:1]: Empty and lonely. I actually don't know what focus is supposed to do.

File: src/components/badge/badge.spec.ts (hunk 2 above)
Comment (yinonov) [reply src/components/badge/badge.spec.ts]: reverted

**Review by YonatanKra — APPROVED** [review]

File: src/components/focus/README.md (hunk 3 above)
Comment (yinonov) [reply src/components/focus/README.md:1]: opened a ticket for this

**Pull request merged**